plugins/octant/go.mod

```diff
@@ -4,14 +4,17 @@ go 1.15
 
 require (
 	github.com/vmware-tanzu/antrea v0.0.0
-	github.com/vmware-tanzu/octant v0.16.1
-	k8s.io/apimachinery v0.19.0-beta.2
-	k8s.io/client-go v0.19.0-alpha.3
+	github.com/vmware-tanzu/octant v0.17.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
 )
 
 replace (
 	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20201109024835-6fd225d8c8d1
 	github.com/vmware-tanzu/antrea => ../../
+	k8s.io/api => k8s.io/api v0.19.8
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.8
+	k8s.io/client-go => k8s.io/client-go v0.19.8
```

Comment:
These redirection are needed as octant has not updated to support k8s 1.21. Otherwise, we will hit errors such as
```
github.com/vmware-tanzu/antrea/plugins/octant/cmd/antrea-octant-plugin imports
        github.com/vmware-tanzu/octant/pkg/plugin imports
        github.com/vmware-tanzu/octant/internal/module tested by
        github.com/vmware-tanzu/octant/internal/module.test imports
        github.com/vmware-tanzu/octant/internal/cluster/fake imports
        k8s.io/client-go/kubernetes/typed/batch/v2alpha1: module k8s.io/client-go@latest found (v0.21.0), but does not contain package k8s.io/client-go/kubernetes/typed/batch/v2alpha1
```

Reply:
thanks for explanation